app/assets/javascripts/application.js

```diff
@@ -123,6 +123,7 @@ function onContentLoad(){
       if ($(this).data('on-complete')){
         window[$(this).data('on-complete')].call(null, this, status);
       }
+      $(document).trigger('ContentLoad');
```

Comment:
Take a look how I do it in Logs patch, I gave up on the attribute and wrote my own initialization, because my table gets totally different configuration. There is nothing commin, except the positioning of Filter drop down...